about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <title>Women in Tech</title>
+</head>
+<body>
+  <main>
+    <!-- Header Starts Here -->
+    <header>
+     <!-- Top Section -->
+     <div class="top-section">
+      <ul class="top-section-ul">
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+     </div>
+ 
+     <!-- Navbar -->
+     <!--Mobile-->
+     <nav class="navbar-mobile">
+       <div class="menu">
+         <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="menu"> </a>
+          </div>
+     </nav>
+
+     <!-- Nav Items -->
+     <nav class="navbar-container">
+      <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+      <ul class="navbar-items">
+        <li>
+          <a href="about.html" class="menu-link-desktop">About</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Program</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Join</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Sponsor</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">News</a>
+        </li>
+
+        <li class="list-items">
+          <a href="#">WITMP 2023</a>
+        </li>
+      </ul>
+    </nav>
+</header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="about-theme-info">
+<!-- Hero -->
+<div class="about-first-section">
+<div>
+<h4 class="about-theme-intro">"Join us this year!"</h4>
+<h2 class="about-theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
+</div>
+
+<div class="about-intro">
+<h4 class="about-intro-text">
+Are you a woman trying to navigate the tech space? This mentorship program for women in tech contribute to the retention and advancement of female talent, as mentees gain confidence, expand their professional networks, and acquire the necessary skills to excel in their careers, leading to increased diversity and innovation in the tech sector. <span class="button-span">Click the button below to register now.</span>
+</h4>
+<a href="#" class="button-a">
+  <button class="button"> Click Here to Register  </button></a>
+</div>
+</div>
+
+<div class="about-second-section">
+<div class="about-program">
```

Comment:
- [ ] nice work so far. please make the main text bolder and bigger and add some padding to it. like in the original design.  please pay attention to margins and paddings.
- [ ] the text following it has its own background color. Please keep this small details as it will make you a great designer.
- [ ] below it is a contact text and an underline email. Please add those small details. 
- [ ] kindly remove the register button in the about page. You can add this later on when modifying it according to your taste. Here, the guidelines want to see how you can recreate a design. Nice work.
- [ ] You are missing the middle section which has a heading, text and logo. Please kindly add that, so the original design can be matched.

yours
![Screenshot (1160)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/2194a2e1-f293-4699-b8ad-b90418fac02e)
original
![Screenshot (1161)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/c175dfa6-9189-43fd-90a1-d31b5c2f8c0c)

Reply:
Thank you so much for your review, I have fixed the layout and added the middle section.
![middle-card](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/111312788/d9968e1e-2669-43b5-8de0-7f8657490d30)